### PR TITLE
Create user page profile

### PR DESCRIPTION
**Pull request by KoichaDev**
This is the complete desktop version. I am aware I need to fix it to be responsive for the phone device. That will be another branch. If you can confirm it's all fine. Can you please merge it on the master, and I will continue the branch from there?

### CODE REVIEW

File: frontend/src/components/Profile/Description.component.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+const description = {
+  background: '#F2F5F7',
+  textAlign: 'left',
+  padding: '1rem 0'
+};
+
+export default function Description({ txt }) {
+  return (
+    <Fragment>
```
Comment (hbendix) [inline frontend/src/components/Profile/Description.component.jsx]: Unessecary fragment, can just return the p tag

File: frontend/src/components/Profile/Body.component.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  borderTop: {
+    borderTop: '1px solid grey'
+  }
+}));
+
+function Body({ txt, icon, label }) {
+  const classes = useStyles();
+  return (
+    <main className={classes.grid}>
+      <div></div>
```
Comment (hbendix) [inline frontend/src/components/Profile/Body.component.jsx]: Just curious why the empty divs, for this component and others?

File: frontend/src/components/Profile/Description.component.jsx (hunk 1 above)
Comment (KoichaDev) [reply frontend/src/components/Profile/Description.component.jsx]: Very true!

**Comment by KoichaDev** [comment]
No worries! I am just happy I can contribute something useful for people who need this system up and running. 

Thanks for letting me know. I will try my best, and be more careful and study the Material UI Grid system for the assignment.

**Pull request merged**